setup.py

```diff
@@ -192,6 +192,9 @@ def additional_commands():
                 'qt = traitsui.qt4:toolkit',
                 'null = traitsui.null:toolkit',
             ],
+            'etsdemo_data': [
+                'demo = traitsui.extras._demo_info:info',
```

Comment:
IIUC, we aren't using the LHS of the entry point (`demo = ...`), only the RHS, which is what we're calling to load the resources. I'm not sure what other projects do with the LHS. Do they just ignore it?

Reply:
The LHS is not used by etsdemo. Whether the project defining the entry points wants to use LHS is up to the projects I guess.